pmm-app/src/pmm-qan/panel/components/Details/Details.styles.ts

```diff
@@ -1,7 +1,52 @@
 import { css } from 'emotion';
 
-export const styles = {
-  zeroMargin: css`
+import { stylesFactory } from '@grafana/ui';
+import { GrafanaTheme } from '@grafana/data';
+import { getThemeParameters } from 'shared/components/helpers/selectThemeVariant';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const parameters = getThemeParameters(theme);
+
+  return {
+    zeroMargin: css`
     margin: 0 !important;
   `,
-};
+    detailsGrid: css`
+      background-color: ${parameters.table.backgroundColor};
+      display: grid;
```

Comment:
do we need the grid here?

Reply:
Good point, I think that's a legacy code and It need to be cleaned up, in this PR I didn't add it, just copied from scss to styles.ts file